Client/src/Pages/Maintenance/CreateMaintenanceWindow/index.jsx

```diff
@@ -45,9 +46,27 @@ const configOptionTitle = (title, description) => {
   );
 };
 
+const timeFrames = [
```

Comment:
These are mapped to a select called `duration`, I think it would make more sense to call this `durations`?  Or `durationOptions` or something related to that.

Reply:
I meant the `timeFrames` object, it would make more sense for it to be named `durationOptions` or `durations` instead of `timeFrames`